src/Components/Character.js

```diff
@@ -0,0 +1,37 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+function Character({ character }) {
+  return (
+    <li className="character-item" key={character.id}>
+      <img src={character.image} alt={character.name} className="character-image" />
+      <div className="character-container">
+        <div className="card-description d-flex-col">
+          <div>
+            Name:
+            {' '}
+            {character.name}
+            <br />
+            Status:
+            {' '}
+            {character.status}
+          </div>
+          <div className="card-button">
+            <Link to={`/character/${character.id}`}>More</Link>
+          </div>
+        </div>
+      </div>
+    </li>
+  );
+}
```

Comment:
[OPTIONAL] I think you can destructure the object character. e.g. `const {id, image, name, status} = character`. This is considered a JS Best Practice and will make your code look cleaner. You can apply this where you consider it necessary.